estate/models/estate_property.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# licence
+
+from odoo import api
+from odoo import fields
+from odoo import models
+from odoo.exceptions import UserError
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = "Estate property"
+
+    name = fields.Char("Title", required=True, translate=True)
+    description = fields.Text("Property description")
+    postcode = fields.Char("Postcode")
+    date_availability = fields.Date("Date of availability", default=lambda _self: fields.Date.add(fields.Date.today(), days=30), copy=False)
+    expected_price = fields.Float("Expected price", required=True)
+    selling_price = fields.Float("Selling price", readonly=True, copy=False)
+    bedrooms = fields.Integer("No. of bedrooms", default=2)
+    living_area = fields.Integer("Living area (sqm)")
+    facades = fields.Integer("No. of facades")
+    garage = fields.Boolean("Has a garage")
+    garden = fields.Boolean("Has a garden", default=False)
+    garden_area = fields.Integer("Garden area (sqm)", default=0)
+    garden_orientation = fields.Selection(
+        string="Garden orientation",
+        selection=[
+            ('north', "North"),
+            ('south', "South"),
+            ('east', "East"),
+            ('west', "West")
+            ],
+        default=None,
+        )
```

Comment:
```suggestion
    garden_orientation = fields.Selection(
        string="Garden orientation",
        selection=[
            ('north', "North"),
            ('south', "South"),
            ('east', "East"),
            ('west', "West"),
        ],
        default=None,
    )
```
It's a good practice to include a trailing comma in multi-line lists or tuples. This helps preserve blames (the history of what commit changed which line). This can also reduce diff size when parameters or elements are added or removed from function calls, function definitions, literals, etc.